stories/Chessboard.stories.tsx

```diff
@@ -446,3 +447,40 @@ export const StyledBoard = () => {
     </div>
   );
 };
+
+///////////////////////////////////
+////////// Custom Square ///////////
+///////////////////////////////////
+const CustomSquareRenderer = forwardRef<HTMLDivElement, CustomSquareProps>((props, ref) => {
+  const { children, square, squareColor, style } = props;
+
+  return (
+    <div ref={ref} style={{ ...style, position: "relative" }}>
+      {children}
+      <div
+        style={{
+          position: "absolute",
+          right: 0,
+          bottom: 0,
+          display: "flex",
+          alignItems: "center",
+          justifyContent: "center",
+          height: 16,
+          width: 16,
+          borderTopLeftRadius: 6,
+          backgroundColor: squareColor === "black" ? "#064e3b" : "#312e81",
+          color: "#fff",
+          fontSize: 14,
+        }}
+      >
+        {square}
+      </div>
+    </div>
+  );
+});
+
+export const CustomSquare = Template.bind({});
+CustomSquare.args = {
+  id: "Custom Square",
+  customSquare: CustomSquareRenderer
+};
```

Comment:
can this story be written similarly to the above ones so that the example code is visible in the docs please?

Reply:
Sure. `CustomSquareRenderer` is a component type, so I've left a comment within the story of this component definition so that it is visible within docs.